### ISSUE TITLE
Re-enable redis tests for 1.1 branch

### ISSUE THREAD

**Issue by mandarjog** [istio/istio#14424]
Redis tests fail on 1.1 branch due to Tiller and redis helm chart errors.
This has been fixed in the master.

Disabling redis tests to get 1.1 merging PRs again.
Fix this and re-enable tests in 1.1

```
Error: cannot connect to Tiller
2019-05-24T16:47:58.404645Z    info    Command error: exit status 1
2019-05-24T16:48:08.405087Z    info    Running command helm version
2019-05-24T16:48:08.629759Z    info    Command output: 
Client: &version.Version{SemVer:"v2.7.2", GitCommit:"8478fb4fc723885b155c924d1c8c410b7a9444e6", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.7.2", GitCommit:"8478fb4fc723885b155c924d1c8c410b7a9444e6", GitTreeState:"clean"}
2019-05-24T16:48:08.629839Z    info    Tiller is running
2019-05-24T16:48:08.629855Z    info    Running command helm install stable/redis --name redis-release --namespace istio-system --set usePassword=false,persistence.enabled=false
2019-05-24T16:48:10.361863Z    info    Command output: 
Error: error validating "": error validating data: field spec.volumeClaimTemplates[0].spec.dataSource for v1.PersistentVolumeClaimSpec is required
2019-05-24T16:48:10.361949Z    info    Command error: exit status 1
2019-05-24T16:48:10.361974Z    error    Failed to complete Init. Error helm install stable/redis failed, setValue=--set usePassword=false,persistence.enabled=false
2019-05-24T16:48:10.361979Z    info    Saving logs
2019-05-24T16:48:10.361985Z    info    Creating status file
```

**Comment by mandarjog** [comment]
@kyessenov

**Comment by howardjohn** [comment]
Fixed by #14497
